upup/models/cloudup/resources/addons/networking.projectcalico.org.canal/k8s-1.16.yaml.template

```diff
@@ -780,7 +797,7 @@ spec:
             privileged: true
           resources:
             requests:
-              cpu: 250m
+              cpu: 90m
```

Comment:
This is still `250m` upstream as far as I can see, any reason to lower it?

Reply:
https://github.com/kubernetes/kops/commit/ba76db7f41e7584d156f845197e68308a9050895

Reply:
Aha thanks for linking, okay sounds good 👍